### PR TITLE
add Python question

### PR DESCRIPTION
**Pull request by Wendell00**
## Summary of changes

Add Python question, about random library

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1632,6 +1632,16 @@ const pythonQuiz = [
...
+  },
+  {
+    Question: "Which of the following statements is true regarding the Python random library?",
+    Answer: "The random library can only generate pseudo-random numbers.",
+    Distractor1: "The random library can generate true random numbers based on hardware entropy.",
+    Distractor2: "The random library is primarily used for generating random strings.",
+    Distractor3: "The random library cannot be used to generate random numbers within a specified range.",
+    Explanation:
+      "The random module in Python generates pseudo-random numbers, which are generated by mathematical algorithms and are not truly random but are sufficiently unpredictable for many purposes. To generate truly random numbers, you may need specialized hardware or external services, which are not provided by the random module.",
+    Link: "https://www.geeksforgeeks.org/python-random-module/"
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1644]: we only accept links from freeCodeCamp resources, wikipedia or official docs

```suggestion
    Link: "https://docs.python.org/3/library/random.html"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Wendell00** [comment]
Ok, i commit

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**